libs/langchain/langchain/agents/openai_functions_agent/base.py

```diff
@@ -109,7 +109,9 @@ def _parse_ai_message(message: BaseMessage) -> Union[AgentAction, AgentFinish]:
     if function_call:
         function_name = function_call["name"]
         try:
-            _tool_input = json.loads(function_call["arguments"])
+            # escape special characters
+            _tool_input = json.dumps(function_call["arguments"])
+            _tool_input = json.loads(_tool_input)
```

Comment:
could you check if this works for you?

```python
_tool_input = json.loads(function_call["arguments"], strict=False)
```

Reply:
@eyurtsev this approach doesn't fix the issue btw